src/libraries/sendtohelixhelp.proj

```diff
@@ -233,13 +232,32 @@
     </ItemGroup>
   </Target>
 
+  <PropertyGroup Condition="'$(IsXUnitLogCheckerSupported)' == 'true' and '$(BuildTargetFramework)' == '$(NetCoreAppCurrent)'">
+    <_ShouldSetXUnitLogCheckerEnvVar>1</_ShouldSetXUnitLogCheckerEnvVar>
+    <HelixPreCommands Condition="'$(TargetOS)' == 'windows'">
+      $(HelixPreCommands);
+      set __TestArchitecture=$(TargetArchitecture)
+      set __IsXUnitLogCheckerSupported=$(_ShouldSetXUnitLogCheckerEnvVar)
+    </HelixPreCommands>
+    <HelixPreCommands Condition="'$(TargetOS)' != 'windows'">
+      $(HelixPreCommands);
+      export __TestArchitecture=$(TargetArchitecture)
```

Comment:
These don't need semicolons between the lines?

Reply:
I am following the same pattern as the other similar examples in this file.

Reply:
Ah, yes, missed semicolon
```suggestion
      export __TestArchitecture=$(TargetArchitecture);
```